tests/metrics/test_mean_absolute_error.py

```diff
@@ -0,0 +1,37 @@
+import jax.numpy as jnp
+import numpy as np
+
+import torch
+from torchmetrics import MeanAbsoluteError
+
+from treex.metrics.mean_absolute_error import MeanAbsolutError
```

Comment:
When the typo is fixed these classes will have the same name, probably best to do something like:

```python
import torchmetrics as tm
import treex as tx
```
and later use them as:

```python
tm.MeanAbsoluteError(...)
tx.metrics.MeanAbsoluteError(...)
```